addons/mail/static/src/components/attachment_viewer/attachment_viewer.js

```diff
@@ -157,11 +167,10 @@ class AttachmentViewer extends Component {
      */
     _handleImageLoad() {
         if (
-            this.attachmentViewer.attachment.fileType === 'image' &&
-            this._renderedAttachment !== this.attachmentViewer.attachment
```

Comment:
maybe revert condition change

Reply:
Or another alternative would be if we can detect if the image is already loaded or not, and add listener only if image is not loaded (and then we can still remove the other condition)